src/rtc_time.hpp

```diff
@@ -283,6 +283,39 @@ struct RTCTime {
      * @return unsigned long long The delta between the two structs in seconds.
      */
     long long operator-(const RTCTime &rhs) const;
```

Comment:
This should return an RCTTime struct?
If thats not the case, make it an unsigned long to avoid type conversion